src/main/java/org/veupathdb/service/eda/ds/plugin/standalonemap/CollectionFloatingBoxplotPlugin.java

```diff
@@ -0,0 +1,126 @@
+package org.veupathdb.service.eda.ds.plugin.standalonemap;
+
+import org.gusdb.fgputil.ListBuilder;
+import org.gusdb.fgputil.validation.ValidationException;
+import org.veupathdb.service.eda.common.client.spec.StreamSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.ConstraintSpec;
+import org.veupathdb.service.eda.common.plugin.constraint.DataElementSet;
+import org.veupathdb.service.eda.common.plugin.util.PluginUtil;
+import org.veupathdb.service.eda.common.plugin.util.RFileSetProcessor;
+import org.veupathdb.service.eda.ds.Resources;
+import org.veupathdb.service.eda.ds.plugin.AbstractEmptyComputePlugin;
+import org.veupathdb.service.eda.ds.plugin.AbstractPlugin;
+import org.veupathdb.service.eda.ds.plugin.standalonemap.markers.OverlaySpecification;
+import org.veupathdb.service.eda.generated.model.*;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static org.veupathdb.service.eda.common.plugin.util.RServeClient.streamResult;
+import static org.veupathdb.service.eda.common.plugin.util.RServeClient.useRConnectionWithProcessedRemoteFiles;
+import static org.veupathdb.service.eda.ds.metadata.AppsMetadata.VECTORBASE_PROJECT;
+
+public class CollectionFloatingBoxplotPlugin extends AbstractEmptyComputePlugin<FloatingBoxplotPostRequest, FloatingBoxplotSpec> {
+  private OverlaySpecification _overlaySpecification = null;
+
+  @Override
+  public String getDisplayName() {
+    return "Box plot";
+  }
+
+  @Override
+  public String getDescription() {
+    return "Visualize summary values for a continuous Variable Group";
```

Comment:
are they grouped variables or a variable group? Whichever is fine just noting that we're not consistent yet

Reply:
No idea lol